packages/convenience/test/Convenience.sol.js

```diff
@@ -1,46 +1,259 @@
 const { expect } = require("chai");
```

Comment:
The file is not prettified & linted. Husky is supposed to do this automatically but it's broken at the moment, will have to do that manually at the project root.